packages/components/src/clipboard-button/README.md

```diff
@@ -24,3 +24,35 @@ const MyClipboardButton = () => {
 	);
 };
 ```
+
+## Props
+
+The component accepts the following props:
```

Comment:
Would it make sense to add something like: All the other properties not referred here are passed as properties to the button component?

Reply:
Sorry @jorgefilipecosta , I didn't see your comment and I created a [new conversation](https://github.com/WordPress/gutenberg/pull/34711#discussion_r708359567)